[2023-11-21T20:20:46.797+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projet_IDD.load_data manual__2023-11-21T20:20:42.099001+00:00 [queued]>
[2023-11-21T20:20:46.811+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projet_IDD.load_data manual__2023-11-21T20:20:42.099001+00:00 [queued]>
[2023-11-21T20:20:46.812+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-21T20:20:46.833+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-21 20:20:42.099001+00:00
[2023-11-21T20:20:46.841+0000] {standard_task_runner.py:57} INFO - Started process 3605 to run task
[2023-11-21T20:20:46.845+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Projet_IDD', 'load_data', 'manual__2023-11-21T20:20:42.099001+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpedi3tsi1']
[2023-11-21T20:20:46.849+0000] {standard_task_runner.py:85} INFO - Job 108: Subtask load_data
[2023-11-21T20:20:46.928+0000] {task_command.py:416} INFO - Running <TaskInstance: Projet_IDD.load_data manual__2023-11-21T20:20:42.099001+00:00 [running]> on host 726cce66553f
[2023-11-21T20:20:47.218+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projet_IDD' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-21T20:20:42.099001+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-21T20:20:42.099001+00:00'
[2023-11-21T20:20:47.380+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-21T20:20:48.336+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1890, in _execute_context
    self.dialect.do_executemany(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 982, in do_executemany
    context._psycopg2_fetched_rows = xtras.execute_values(
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.ForeignKeyViolation: insert or update on table "passages_corona" violates foreign key constraint "passages_corona_dep_fkey"
DETAIL:  Key (dep)=(1) is not present in table "departements".


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ETL.py", line 57, in load_data
    df_urgences.to_sql('passages_corona', postgres_sql_upload.get_sqlalchemy_engine(), if_exists='append', chunksize=1000, index=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/generic.py", line 2878, in to_sql
    return sql.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 769, in to_sql
    return pandas_sql.to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1920, in to_sql
    total_inserted = sql_engine.insert_records(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1470, in insert_records
    raise err
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1461, in insert_records
    return table.insert(chunksize=chunksize, method=method)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 1023, in insert
    num_inserted = exec_insert(conn, keys, chunk_iter)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/sql.py", line 929, in _execute_insert
    result = conn.execute(self.table.insert(), data)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1385, in execute
    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 334, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1577, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1953, in _execute_context
    self._handle_dbapi_exception(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2134, in _handle_dbapi_exception
    util.raise_(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1890, in _execute_context
    self.dialect.do_executemany(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 982, in do_executemany
    context._psycopg2_fetched_rows = xtras.execute_values(
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
sqlalchemy.exc.IntegrityError: (psycopg2.errors.ForeignKeyViolation) insert or update on table "passages_corona" violates foreign key constraint "passages_corona_dep_fkey"
DETAIL:  Key (dep)=(1) is not present in table "departements".

[SQL: INSERT INTO passages_corona (dep, date_de_passage, sursaud_cl_age_corona, nbre_pass_corona, nbre_pass_tot, nbre_hospit_corona, nbre_pass_corona_h, nbre_pass_corona_f, nbre_pass_tot_h, nbre_pass_tot_f, nbre_hospit_corona_h, nbre_hospit_corona_f, nbre_acte_corona, nbre_acte_tot, nbre_acte_corona_h, nbre_acte_corona_f, nbre_acte_tot_h, nbre_acte_tot_f) VALUES (%(dep)s, %(date_de_passage)s, %(sursaud_cl_age_corona)s, %(nbre_pass_corona)s, %(nbre_pass_tot)s, %(nbre_hospit_corona)s, %(nbre_pass_corona_h)s, %(nbre_pass_corona_f)s, %(nbre_pass_tot_h)s, %(nbre_pass_tot_f)s, %(nbre_hospit_corona_h)s, %(nbre_hospit_corona_f)s, %(nbre_acte_corona)s, %(nbre_acte_tot)s, %(nbre_acte_corona_h)s, %(nbre_acte_corona_f)s, %(nbre_acte_tot_h)s, %(nbre_acte_tot_f)s)]
[parameters: ({'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 9.0, 'nbre_pass_tot': 435.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': 6.0, 'nbre_pass_corona_f': 3.0, 'nbre_pass_tot_h': 219.0, 'nbre_pass_tot_f': 216.0, 'nbre_hospit_corona_h': 1.0, 'nbre_hospit_corona_f': 3.0, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 58.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 30.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 3, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 138.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 88.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 45.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 6, 'nbre_pass_corona': 6.0, 'nbre_pass_tot': 75.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 27, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 6.0, 'nbre_pass_tot': 452.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': 1.0, 'nbre_pass_corona_f': 5.0, 'nbre_pass_tot_h': 236.0, 'nbre_pass_tot_f': 216.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 4.0, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}  ... displaying 10 of 1000 total bound parameter sets ...  {'dep': 1, 'date_de_passage': datetime.datetime(2023, 5, 17, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 3.0, 'nbre_pass_tot': 84.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2023, 5, 17, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 40.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None})]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
[2023-11-21T20:20:48.386+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Projet_IDD, task_id=load_data, execution_date=20231121T202042, start_date=20231121T202046, end_date=20231121T202048
[2023-11-21T20:20:48.410+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 108 for task load_data ((psycopg2.errors.ForeignKeyViolation) insert or update on table "passages_corona" violates foreign key constraint "passages_corona_dep_fkey"
DETAIL:  Key (dep)=(1) is not present in table "departements".

[SQL: INSERT INTO passages_corona (dep, date_de_passage, sursaud_cl_age_corona, nbre_pass_corona, nbre_pass_tot, nbre_hospit_corona, nbre_pass_corona_h, nbre_pass_corona_f, nbre_pass_tot_h, nbre_pass_tot_f, nbre_hospit_corona_h, nbre_hospit_corona_f, nbre_acte_corona, nbre_acte_tot, nbre_acte_corona_h, nbre_acte_corona_f, nbre_acte_tot_h, nbre_acte_tot_f) VALUES (%(dep)s, %(date_de_passage)s, %(sursaud_cl_age_corona)s, %(nbre_pass_corona)s, %(nbre_pass_tot)s, %(nbre_hospit_corona)s, %(nbre_pass_corona_h)s, %(nbre_pass_corona_f)s, %(nbre_pass_tot_h)s, %(nbre_pass_tot_f)s, %(nbre_hospit_corona_h)s, %(nbre_hospit_corona_f)s, %(nbre_acte_corona)s, %(nbre_acte_tot)s, %(nbre_acte_corona_h)s, %(nbre_acte_corona_f)s, %(nbre_acte_tot_h)s, %(nbre_acte_tot_f)s)]
[parameters: ({'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 9.0, 'nbre_pass_tot': 435.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': 6.0, 'nbre_pass_corona_f': 3.0, 'nbre_pass_tot_h': 219.0, 'nbre_pass_tot_f': 216.0, 'nbre_hospit_corona_h': 1.0, 'nbre_hospit_corona_f': 3.0, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 1, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 58.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 2, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 30.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 3, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 138.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 88.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 1.0, 'nbre_pass_tot': 45.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 26, 0, 0), 'sursaud_cl_age_corona': 6, 'nbre_pass_corona': 6.0, 'nbre_pass_tot': 75.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2022, 12, 27, 0, 0), 'sursaud_cl_age_corona': 0, 'nbre_pass_corona': 6.0, 'nbre_pass_tot': 452.0, 'nbre_hospit_corona': 4.0, 'nbre_pass_corona_h': 1.0, 'nbre_pass_corona_f': 5.0, 'nbre_pass_tot_h': 236.0, 'nbre_pass_tot_f': 216.0, 'nbre_hospit_corona_h': 0.0, 'nbre_hospit_corona_f': 4.0, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}  ... displaying 10 of 1000 total bound parameter sets ...  {'dep': 1, 'date_de_passage': datetime.datetime(2023, 5, 17, 0, 0), 'sursaud_cl_age_corona': 4, 'nbre_pass_corona': 3.0, 'nbre_pass_tot': 84.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None}, {'dep': 1, 'date_de_passage': datetime.datetime(2023, 5, 17, 0, 0), 'sursaud_cl_age_corona': 5, 'nbre_pass_corona': 0.0, 'nbre_pass_tot': 40.0, 'nbre_hospit_corona': 0.0, 'nbre_pass_corona_h': None, 'nbre_pass_corona_f': None, 'nbre_pass_tot_h': None, 'nbre_pass_tot_f': None, 'nbre_hospit_corona_h': None, 'nbre_hospit_corona_f': None, 'nbre_acte_corona': None, 'nbre_acte_tot': None, 'nbre_acte_corona_h': None, 'nbre_acte_corona_f': None, 'nbre_acte_tot_h': None, 'nbre_acte_tot_f': None})]
(Background on this error at: https://sqlalche.me/e/14/gkpj); 3605)
[2023-11-21T20:20:48.466+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-21T20:20:48.486+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
